subsys/mgmt/mcumgr/smp_shell.c

```diff
@@ -136,10 +140,19 @@ static uint16_t smp_shell_get_mtu(const struct net_buf *nb)
 	return CONFIG_MCUMGR_SMP_SHELL_MTU;
 }
 
+extern struct shell_transport shell_transport_uart;
+
 static int smp_shell_tx_raw(const void *data, int len, void *arg)
 {
-	/* Cast away const. */
-	k_str_out((void *)data, len);
+	struct shell_uart *const su = shell_transport_uart.ctx;
+	struct shell_uart_ctrl_blk *const scb = su->ctrl_blk;
```

Comment:
I think we can also add `const` here as well

Reply:
Done